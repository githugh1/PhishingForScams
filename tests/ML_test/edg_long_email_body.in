{
  "email_subject": "Long Email Subject",
  "email_body": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. ... (long text)",
  "sender": "longemail@email.com"
}
