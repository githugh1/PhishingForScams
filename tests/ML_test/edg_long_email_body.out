{
  "is_spam": true,
  "confidence": 0.8,
  "message": "This email is likely spam."
}
